usr/src/uts/common/sys/cpuvar.h

```diff
@@ -610,8 +610,6 @@ int	cpu_kstat_intrstat_update(kstat_t *, int);
 void	cpu_intr_swtch_enter(kthread_t *);
 void	cpu_intr_swtch_exit(kthread_t *);
 
-void	mbox_lock_init(void);	 /* initialize cross-call locks */
```

Comment:
I'm going to try to remember to go through a bunch of these headers, and delete the copy/paste damage.  Problem is intel has the damage too, so I'd feel like I'd need to do it upstream

Reply:
see main comment